spec/lib/efile/md/md502_su_calculator_spec.rb

```diff
@@ -29,11 +29,90 @@
     end
   end
 
+  describe "#calculate_military_per_filer" do
+    context "when the taxable amount is higher than the age calculation" do
```

Comment:
[dust] small description change -- same for the lower scenario
```suggestion
    context "when the taxable amount is higher than the age benefit amount" do
```